### PR TITLE
added Bouachain Mainet

### PR DESCRIPTION
**Pull request by Bouachain**

### CODE REVIEW

File: bouachain/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "description": "The native token of the Bouachain blockchain, used for transaction fees, staking, and governance.",
+      "denom_units": [
+        {
+          "denom": "uBOUA",
+          "exponent": 0
+        }
+	],      
+      "base": "bouacoin",
+      "name": "BOUACOIN",
+      "display": "BOUA",
```
Comment (JeremyParish69) [inline bouachain/assetlist.json:15]: This display denom is not defined under denom_units, but must be. Usually this is used to inform the number of decimals/exponents.

File: bouachain/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bouachain/images/bouachain.svg",
+          "theme": {
+            "primary_color_hex": "#FF00C1"
```
Comment (JeremyParish69) [inline bouachain/assetlist.json:26]: Where does this color come from?

File: bouachain/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "assets": [
+    {
+      "description": "The native token of the Bouachain blockchain, used for transaction fees, staking, and governance.",
+      "denom_units": [
+        {
+          "denom": "uBOUA",
+          "exponent": 0
+        }
+	],      
+      "base": "bouacoin",
```
Comment (JeremyParish69) [inline bouachain/assetlist.json:13]: This display denom is not defined under denom_units, but must be. It must match the denon_unit with 0 exponent.

File: bouachain/chain.json [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        "time": "1814400s"
+      }
+    },
+    "codebase": {
+      "git_repo": "https://github.com/Bouachain/bouachain.git",
+      "recommended_version": "v1.0.0",
+      "compatible_versions": [
+        "v0.9.0"
+      ],
+      "cosmos_sdk_version": "0.50.8",
```
Comment (JeremyParish69) [inline bouachain/chain.json:44]: Will need the 'sdk' property

File: bouachain/chain.json [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+      "consensus": {
+        "type": "tendermint",
+        "version": "0.38.10"
+      },
+      "cosmwasm_version": "0.50.0",
+      "cosmwasm_enabled": true,
+      "ibc_go_version": "8.4.0",
```
Comment (JeremyParish69) [inline bouachain/chain.json:51]: Will need the 'ibc' property

File: bouachain/chain.json [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+      "recommended_version": "v1.0.0",
+      "compatible_versions": [
+        "v0.9.0"
+      ],
+      "cosmos_sdk_version": "0.50.8",
+      "consensus": {
+        "type": "tendermint",
+        "version": "0.38.10"
+      },
+      "cosmwasm_version": "0.50.0",
```
Comment (JeremyParish69) [inline bouachain/chain.json:49]: Will need the 'cosmwasm' property

File: bouachain/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply bouachain/assetlist.json:26]: Seems like it should be #000000

**Comment by Bouachain** [comment]
Fixed, Please conduct a new review.

File: bouachain/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "bouacoin",
+            "exponent": 8
+          }
+      ],      
+        "base": "ubouacoin",
+        "name": "BOUACOIN",
+        "display": "BOUA",
```
Comment (JeremyParish69) [inline bouachain/assetlist.json:19]: still, the dispaly denom "BOUA" is not defined

Comment (JeremyParish69) [reply bouachain/assetlist.json:19]: likely looking for "bouacoin" since that is defined as exp 8

**Comment by Bouachain** [comment]
Fixed ✅✅

File: bouachain/chain.json [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+     "versions": [
+      {
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+        "v0.9.0"
+      ],
+        "sdk": {
+          "type": "cosmos",
+          "repo": "github.com/rollchains/cosmos-sdk",
```
Comment (JeremyParish69) [inline bouachain/chain.json:57]: This must have "https://" at the beginning.

File: bouachain/chain.json [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+          "version": "v0.50.8",
+          "tag": "v0.50.8"
+        }
+      }
+    ],
```
Comment (JeremyParish69) [inline bouachain/chain.json:62]: Schema is invalid. I think it might be that you're not closing the 'codebase' object.

**Comment by Bouachain** [comment]
Added Https and closed the code base

Fixed ✅✅

File: bouachain/assetlist.json [hunk 10]
```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "bouachain",
+    "assets": [
+      {
```
Comment (JeremyParish69) [inline bouachain/assetlist.json:5]: type_asset not specified

Comment (Bouachain) [reply bouachain/assetlist.json:5]: Ok let me fix that now

File: bouachain/chain.json [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+        }
+      ],
+      "rest": [
+        {
+          "address": "https://lcd.bouachain.com",
+          "provider": "BouaValidator"
+        }
+      ]
+    },
+    "grpc": [
```
Comment (JeremyParish69) [inline bouachain/chain.json:84]: grpc should be inside apis

**Comment by Bouachain** [comment]
Fixed ✅😄

**Pull request merged**